Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserModel.h

```diff
@@ -84,10 +85,14 @@ namespace AzToolsFramework
             AZStd::shared_ptr<RootAssetBrowserEntry> GetRootEntry() const;
             void SetRootEntry(AZStd::shared_ptr<RootAssetBrowserEntry> rootEntry);
 
+            AssetBrowserFilterModel* GetFilterModel() const;
+            void setFilterModel(AssetBrowserFilterModel* filterModel);
+
             static void SourceIndexesToAssetIds(const QModelIndexList& indexes, AZStd::vector<AZ::Data::AssetId>& assetIds);
             static void SourceIndexesToAssetDatabaseEntries(const QModelIndexList& indexes, AZStd::vector<AssetBrowserEntry*>& entries);
             
         private:
+            AssetBrowserFilterModel* m_filterModel;
```

Comment:
suggestion: A comment to mention this is a 'non owning/view' pointer might be helpful here (and initialize to `nullptr` for good measure)